.scripts/process_build.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+# Sorry, everyone
+if (( ${BASH_VERSION%%.*} < 4 )); then
+  echo "This script requires Bash >= 4. On OSX, try: brew install bash"
+  exit 1
+fi
+
+# Any error is fatal.
+set -e
+set -o pipefail
+
+BUILD_DIR=$(pwd)
```

Comment:
If this is launched via the CI script, do we still need that logic of sourcing bash_profile?